src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/_print.scss

```diff
@@ -0,0 +1,26 @@
+@media screen {
+    .print-only {
+        display: none;
+    }
+
+    .screen-only {
+        display: block;
+    }
+}
+
+@media print {
+    .print-only {
+        display: block;
+    }
+
+    .screen-only {
+        display: none;
+    }
```

Comment:
I don't like utility classes like these because they mix up content and presentation (classes should say what something is, not how it should be displayed).